src/paulscode/android/mupen64plusae/PlayMenuActivity.java

```diff
@@ -213,6 +210,22 @@ protected void onResume()
         mPrefs.registerOnSharedPreferenceChangeListener( this );
         mMogaController.onResume();
         refreshViews();
+        
+        if( ACTION_RESUME.equals( action ) )
```

Comment:
Ah, yes.  The player mapping part is on the todo list for an overhaul.  This is fine, I was just thinking it would simplify your life but I think you are right, this is the easy solution until PlayMenuActivity is removed.